.travis.yml

```diff
@@ -0,0 +1,40 @@
+sudo: required
+
+services:
+  - docker
+
+env:
+  SHORT_COMMIT=`echo $TRAVIS_COMMIT | cut -c1-7`
```

Comment:
Consider using `git describe --long --always --tags` which would include the git tag as well